src/main/scala/li/cil/oc/Constants.scala

```diff
@@ -161,7 +161,8 @@ object Constants {
     final val UpgradeContainerTier1 = "upgradeContainer1"
     final val UpgradeContainerTier2 = "upgradeContainer2"
     final val UpgradeContainerTier3 = "upgradeContainer3"
-    final val WirelessNetworkCard = "wlanCard"
+    final val WirelessNetworkCardTier1 = "wlanCard1"
+    final val WirelessNetworkCardTier2 = "wlanCard2"
```

Comment:
Is it allowed to change oredic names (wlanCard -> wlanCard2)? How does this affect pre-existing worlds?

Reply:
Yeah this was already here, I'm assuming it was added when the tier 2 and 3 data cards were added. Pretty much it will remap the old name to the new one without breaking anything. I tested it on 1.7.10 with a lot of cards, both in chests / player inventory and in computers.

Reply:
Does it work with crafting from other mods? I think that's what the OreDic is for?

Reply:
The main issue here is the item id itself, which would normally cause forge to get confused and say the world is missing items. However, OC handles the event for when forge finds an invalid item, and that's where the remaps come into play. To the user, everything is transparent. As for crafting, I will probably make wlanCard into an alias for wlanCard2

Reply:
Alright, that sounds like it should work, though I have NO idea if it will because I haven't actually worked with MC mod code for a very long time. :P